test/parallel/test-fs-write-stream-encoding.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
 const common = require('../common');
 const assert = require('assert');
+const fixtures = require('../common/fixtures');
 const fs = require('fs');
 const path = require('path');
 const stream = require('stream');
 const firstEncoding = 'base64';
 const secondEncoding = 'latin1';
 
-const examplePath = path.join(common.fixturesDir, 'x.txt');
+const examplePath = path.join(fixtures.fixturesDir, 'x.txt');
```

Comment:
This could alternatively be `fixtures.path('x.txt')`.

Reply:
Ah! Great.  That seems better.  I'll tweak that shortly.